packages/icons/manifest.json

```diff
@@ -385,6 +385,21 @@
       "category": "Payment method",
       "size": "24"
     },
+    {
+      "name": "vr",
+      "category": "Payment method",
+      "size": "24"
+    },
+     {
+      "name": "boleto",
+      "category": "Payment method",
+      "size": "24"
+    },
+    {
+      "name": "nfc_payment",
+      "category": "Payment method",
+      "size": "24"
+    },
```

Comment:
I would still agree with Connor to put the nfc icon to the Card Schemes category.